### PR TITLE
Removes reference to the client only integration

### PR DESCRIPTION
**Pull request by cjavilla-stripe**
r? @tmarek-stripe 

- [x] Removed client-only integraion
- [x] Updated readme

### CODE REVIEW

File: client-and-server/server/php/composer.lock [hunk 1]
```diff
@@ -1,75 +0,0 @@
...
-    "content-hash": "66b83a46a4be69d4204be618354c6181",
-    "packages": [
-        {
-            "name": "stripe/stripe-php",
-            "version": "v7.32.0",
-            "source": {
-                "type": "git",
-                "url": "https://github.com/stripe/stripe-php.git",
-                "reference": "edffb6d9370ea84f01faab0f6874b0e049e9df05"
-            },
```
Comment (tmarek-stripe) [inline client-and-server/server/php/composer.lock:16]: Why the change to the composer.lock for the php server?

Comment (cjavilla-stripe) [reply client-and-server/server/php/composer.lock:16]: Good question. I don't think this should've been deleted!

Comment (cjavilla-stripe) [reply client-and-server/server/php/composer.lock:16]: Looks like github is confused. This was moved from `client-and-server/server/php` to `server/php`
